packages/cli/core/src/lib/snapshot/snapshotFactory.ts

```diff
@@ -21,14 +22,15 @@ export class SnapshotFactory {
     await ensureResourcesAccess(client, project.resourceTypes);
     const pathToZip = await project.compressResources();
     const file = readFileSync(pathToZip);
-
+    const blob = new Blob([file], {
+      type: 'application/zip',
+    });
     const computedOptions: CreateFromFileOptions = {
       developerNotes: 'cli-created-from-zip',
     };
 
-    const model = await client.resourceSnapshot.createFromBuffer(
-      file,
-      ResourceSnapshotSupportedFileTypes.ZIP,
+    const model = await client.resourceSnapshot.createFromFile(
+      blob,
```

Comment:
Byeeee special Node hacked method, now we have blob and proper fetch that support it.